examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "356dc2a9",
+   "metadata": {},
+   "source": [
+    "## Embed and Cluster the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fd47c57",
+   "metadata": {},
+   "source": [
+    "Now we can pass a column from this data to another weave `Op` to embed it. This will turn each row in the column `name` into a vector."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43cec2d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "embeddings = openai.openai_embed(data['name'], {\"model\": \"text-embedding-ada-002\"})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6715de4a",
+   "metadata": {},
+   "source": [
+    "Now, we'll project this to 2 dimensions for clustering, and then we'll cluster this projection using `hdbscan`."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "500e19b1",
+   "id": "5eff5bca",
    "metadata": {},
    "outputs": [],
    "source": [
-    "data"
+    "clusterable_projection = umap.umap_project(\n",
+    "    embeddings, {\n",
+    "        'n_neighbors': 30,\n",
+    "        'min_dist': 0,\n",
+    "        'n_components': 2,\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "583830d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clusters = hdbscan.hdbscan_cluster(clusterable_projection, {\n",
+    "            'min_samples': 10,\n",
+    "            'min_cluster_size': 60\n",
+    "        })"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87c4ad19",
+   "metadata": {},
+   "source": [
+    "Then, we'll project the embeddings again for plotting. For efficient computation, weave isn't actually executing anything until weave call `weave.use` or `weave.show` on the built up computation (or \"compute graph\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "21f5917b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "projection = umap.umap_project(embeddings, {})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3b7f76c",
+   "metadata": {},
+   "source": [
+    "## Combine data for plotting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d017836",
+   "metadata": {},
+   "source": [
+    "Finally, we'll combine our data into one big table so we can display the Pokemon name and types alongside our embedding. Here, we're calling `weave.use` on each weave object to execute the computation, and then we're merging them into a new list of dictionarys. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d642110d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "weave.show([{'x': x, 'y': y, 'k':k, 'd':d} for (x,y),k,d in \n",
+    "            zip(weave.use(projection),\n",
+    "                weave.use(clusters),\n",
+    "                weave.use(data))])"
+   ]
+  },
```

Comment:
hmmm, so this doesn't give me a projection—wonder if in the short term, we could walk the reader here through how to make the right PanelPlot step by step...?
<img width="1181" alt="Screen Shot 2023-07-25 at 8 21 24 AM" src="https://github.com/wandb/weave/assets/45573465/2247b190-c95f-422a-9fa6-6a50e1157ce5">

Reply:
the minimal change is pretty small, convert to "Plot" and optionally add the type as the shape (only for fun)?
<img width="1122" alt="Screen Shot 2023-07-25 at 8 27 16 AM" src="https://github.com/wandb/weave/assets/45573465/e91874c1-08af-433f-8421-5baa54e7d1a0">

Reply:
Agreed that we should give better UI instructions.